native/package.json

```diff
@@ -53,8 +53,9 @@
     "@sentry/react-native": "^1.7.2",
     "api-client": "0.0.1",
     "build-configs": "0.0.1",
+    "events": "^3.2.0",
```

Comment:
Somehow needed for htmlparser2 with version >= 5.0. Otherwise the app crashes with `module events not found`